bench_sample.py

```diff
@@ -0,0 +1,34 @@
+import cupy as cp
+import numpy as np
+import pytest
+
+from utils import make_frame
+
+
+@pytest.fixture(params=[0, 1], ids=["AxisIndex", "AxisColumn"])
+def axis(request):
+    return request.param
```

Comment:
I think this fixture will inevitably be useful elsewhere and could go in conftest.